satpy/readers/geos_area.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Geostationary Projection / Area computations.
+
+This module computes properties and area definitions for geostationary
+satellites. It is designed to be a common module that can be called by
+all geostationary satellite readers and uses commonly-included parameters
+such as the CFAC/LFAC values, satellite position, etc, to compute the
+correct area definition.
+"""
+
+import numpy as np
+from pyresample import geometry
+
+
+def get_xy_from_linecol(line, col, offsets, factors):
+    """Get the intermediate coordinates from line & col.
+
+    Intermediate coordinates are actually the instruments scanning angles.
+    """
+    loff, coff = offsets
+    lfac, cfac = factors
+    x__ = float(col - coff) / (float(cfac) / 2**16)
+    y__ = float(line - loff) / (float(lfac) / 2**16)
+
+    return x__, y__
+
+
+def make_ext(ll_x, ur_x, ll_y, ur_y, h):
+    """Create the area extent from computed ll and ur.
+
+    Inputs (all in metres):
```

Comment:
I believe the google docstring standard makes this `Args:` and the arguments below it should not have a `-`. See this example document: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Fixed.